bpf/lib/dbg.h

```diff
@@ -41,6 +41,7 @@ enum {
 	DBG_TO_HOST,
 	DBG_TO_STACK,
 	DBG_PKT_HASH,
+	DBG_RR_SLAVE_SEL,
```

Comment:
This needs to be added to the end of the list, otherwise it will break older Cilium binaries.